AjobK/Athena
# 267 follower count

closes #267 

<br />

 **What is it supposed to do**  
The amount of followers a user has is now visible in the statistics section of the page.

<br />

 **How can we test your branch**  
(Un)follow certain users and refresh the page to see their amount of followers increase (or decrease)

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

No styling changes made

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

No styling changes made

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select atleast two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
